src/index.ts

```diff
@@ -11,6 +11,7 @@ export interface IDockerComposeOptions {
   composeOptions?: string[] | (string | string[])[]
   commandOptions?: string[] | (string | string[])[]
   env?: NodeJS.ProcessEnv
+  callback?: (chunk: any) => void
```

Comment:
I'm not exactly convinced that it should be done using callback function, but it looks as most simple solution.

However:
1. `chunk` should be better defined. Afaik it's an instance of `Buffer`, but it needs confirmation.
2. Second argument should be present to allow recognition from which stream (standard output or error) chunk is coming from.

```suggestion
  callback?: (chunk: Buffer, streamSource: 'stdout'|'stderr') => void
```

Reply:
@Steveb-p +1 for typings.

Reply:
I think we should start with minimal effort and watch the feedback.

Reply:
@Steveb-p  It is indeed a Buffer, fixed